apis/fluentbit/v1alpha2/plugins/output/azure_log_analytics_types.go

```diff
@@ -0,0 +1,58 @@
+package output
+
+import (
+	"fmt"
+
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins"
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins/params"
+)
+
+// +kubebuilder:object:generate:=true
+
+// Azure Log Analytics is the Azure Log Analytics output plugin, allows you to ingest your records into Azure Log Analytics Workspace.
+type AzureLogAnalytics struct {
+	// Customer ID or Workspace ID
+	CustomerID *plugins.Secret `json:"customerID"`
+	// Specify the primary or the secondary client authentication key
+	SharedKey *plugins.Secret `json:"sharedKey"`
+	// Name of the event type.
+	LogType string `json:"logType,omitempty"`
+	// Specify the name of the key where the timestamp is stored.
+	TimeKey string `json:"timeKey,omitempty"`
+	// If set, overrides the timeKey value with the `time-generated-field` HTTP header value.
+	TimeGenerated *bool `json:"timeGenerated,omitempty"`
+}
+
+// Name implement Section() method
+func (_ *AzureLogAnalytics) Name() string {
+	return "azureloganalytics"
```

Comment:
The plugin name should be `azure`.

https://docs.fluentbit.io/manual/pipeline/outputs/azure#configuration-file

Reply:
Resolved in 858c0fabc07643fb6ee144bd94b153d00958936f